docs/mllib-guide.md

```diff
@@ -48,6 +48,7 @@ This lists functionality included in `spark.mllib`, the main MLlib API.
 * [Feature extraction and transformation](mllib-feature-extraction.html)
 * [Frequent pattern mining](mllib-frequent-pattern-mining.html)
   * FP-growth
```

Comment:
Could you also add a link for `FP-growth`? It is no longer the only algorithm under FPM.